### ISSUE TITLE
Re-consider the role attribute of the MultiSelect TagList element

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#4812]
### Re-consider the role attribute of the MultiSelect TagList element

Currently, the TagList element of the MultiSelect widget has a `role` attribute set to `listbox`. Such role, however, would [require the TagList to allow the user to select one or more of its items](https://www.w3.org/TR/2010/WD-wai-aria-20100916/roles#listbox). The TagList itself, however does not allow selection, but it only displays the items selected inside the MultiSelect drop-down element (and the MultiSelect widget itself). Also, as per the WAI-ARIA specification, an element with `role="listbox"` should have children with `role="option"`. The `<li>` elements inside the TagList, however, do not have any `role` set.

Having that said, the TagList rendering should be reconsidered so that the `listbox` role is substituted with another applicable one.
